doc/cephadm/install.rst

```diff
@@ -237,6 +237,98 @@ Single host
 
 To configure a Ceph cluster to run on a single host, use the ``--single-host-defaults`` flag when bootstrapping.
 
+
+
+Hybrid single-host deployment
+-------------------------------------------
+
+This approach elaborates a combination of manual and automated, Ceph cluster deployment on a 
+single-host environment through the ``Cephadm`` tool. 
+
+
+Requirements:
+
+- Python 3
+- Systemd
+- Podman or Docker for running containers
+
+
+Assumptions:
+
+- ``Ceph`` package is installed 
+- ``Cephadm`` is installed 
+
+
+
+Bootstrap:
+
+
+Bootstrap a new cluster with the following instruction. 
+It automatically configures all required keyrings and services. 
+
+.. prompt:: bash #
+
+    cephadm bootstrap --mon-ip $(hostname -I | awk '{print $1}') --mgr-id $(hostname -s) --mon-id $(hostname -s) --allow-fqdn-hostname |& tee ~/cephadm.log 
```

Comment:
I think these parts are covered from the beginning of these docs https://docs.ceph.com/en/latest/cephadm/install/#deploying-a-new-ceph-cluster

Reply:
If you'd like, I think you could document on use cases for single-host deployment